### PR TITLE
Handle moderation queue case for reviewer queues

### LINKED ISSUE #20721: Make moderation review queue consistent with the others
Follow up to #20717 

Moderation reviewer queue is constructed differently from the others as it does not have a table class in utils.py. It's currently treated as an edge case, but it makes sense to treat it like the other queues.

### PR DESCRIPTION
**Pull request by GregSherman**
Fixes #20721 

Make a table object in utils with the attributes required to be consistent with other queues. Put it in the registry so that the mod queue case is handled everywhere else.

### CODE REVIEW

**Comment by GregSherman** [comment]
@diox @eviljeff Requesting review

File: src/olympia/reviewers/utils.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ class AddonQueueTable(tables.Table):
         accessor='_current_version__autoapprovalsummary__score',
     )
     show_count_in_dashboard = True
+    view = None
```
Comment (eviljeff) [inline src/olympia/reviewers/utils.py:69]: Can we define `view = queue` directly here? Or do we come up against circular reference errors?

Comment (GregSherman) [reply src/olympia/reviewers/utils.py:69]: importing the functions from views leads to a total crash so I assume it's circular reference.

Comment (eviljeff) [reply src/olympia/reviewers/utils.py:69]: idea: replace `views.reviewer_tables_registry[queue].view` in urls.py with `getattr(views, reviewer_tables_registry[queue].view_name)()` and change `view` in the table classes to be a string called `view_name`.

(other hacks may be available!)

File: src/olympia/reviewers/views.py [hunk 2]
```diff
@@ -305,6 +306,9 @@ def save_motd(request):
 
 
 def queue(request, tab):
+    # if tab == 'moderated':
+    #     return queue_moderated(request, tab)
+
```
Comment (diox) [inline src/olympia/reviewers/views.py:311]: Can be removed

File: src/olympia/reviewers/utils.py [hunk 3]
```diff
@@ -369,6 +370,15 @@ def get_queryset(cls, request, **kw):
         )
 
 
+class ModerationQueueFields:
```
[the remark this answers is not in the document]
Comment (eviljeff) [reply src/olympia/reviewers/utils.py:373]: agree

**Pull request merged**